pkg/bucketaccessrequest/bucketaccessrequest.go

```diff
@@ -119,7 +131,7 @@ func (b *bucketAccessRequestListener) provisionBucketAccess(ctx context.Context,
 		UID:       sa.ObjectMeta.UID}
 	// bucketaccess.Spec.MintedSecretName - set by the driver
 	bucketaccess.Spec.PolicyActionsConfigMapData, err = util.ReadConfigData(b.kubeClient, bucketAccessClass.PolicyActionsConfigMap)
-	if err != nil {
+	if err != nil && err != util.ErrNilConfigMap {
```

Comment:
Is the CM now optional in the BAC? The KEP says it is required.

Reply:
not sure what you meant, should I not check if CM is empty? Nothing guarantees that the CM is populated, we never catch that any other place.

Reply:
My point is that if the CM ref pointer is nil then we don't return an error here, which, to me, means that BAC.CM is optional. If a BAC admission controller is ensuring the BAC.CM ref field is filled in then we should should not have to check for a nil CM ref.